hw/drivers/sensors/dps368/include/dps368/dps368.h

```diff
@@ -0,0 +1,374 @@
+#define     IFX_DPS368_TMP_COEF_SRCE_REG_POS            7
+
+#define     IFX_DPS368_PSR_TMP_READ_REG_ADDR            0x00
+#define     IFX_DPS368_PSR_TMP_READ_LEN                 6
+
+#define     IFX_DPS368_PRS_CFG_REG_ADDR                 0x06
+#define     IFX_DPS368_PRS_CFG_REG_LEN                  1
+
+#define     IFX_DPS368_TMP_CFG_REG_ADDR                 0x07
+#define     IFX_DPS368_TMP_CFG_REG_LEN                  1
+
+#define     IFX_DPS368_MEAS_CFG_REG_ADDR                0x08
+#define     IFX_DPS368_MEAS_CFG_REG_LEN                 1
+#define		IFX_DPS368_MEAS_CFG_REG_SEN_RDY_VAL			0x40
+#define		IFX_DPS368_MEAS_CFG_REG_COEF_RDY_VAL		0x80
+
+
+#define     IFX_DPS368_CFG_REG_ADDR                     0x09
+#define     IFX_DPS368_CFG_REG_LEN                      1
+
+#define     IFX_DPS368_CFG_TMP_SHIFT_EN_SET_VAL         0x08
+#define     IFX_DPS368_CFG_PRS_SHIFT_EN_SET_VAL         0x04
+
+#define     IFX_DPS368_FIFO_READ_REG_ADDR               0x00
+#define     IFX_DPS368_FIFO_REG_READ_LEN                3
+#define     IFX_DPS368_FIFO_BYTES_PER_ENTRY             3
+
+#define     IFX_DPS368_FIFO_FLUSH_REG_ADDR              0x0C
+#define     IFX_DPS368_FIFO_FLUSH_REG_VAL               0b10000000U
+
+#define     IFX_DPS368_CFG_SPI_MODE_POS                 0
+#define     IFX_DPS368_CFG_SPI_MODE_3_WIRE_VAL          1
+#define     IFX_DPS368_CFG_SPI_MODE_4_WIRE_VAL          0
+
+#define     IFX_DPS368_CFG_FIFO_ENABLE_POS              1
+#define     IFX_DPS368_CFG_FIFO_ENABLE_VAL              1
+#define     IFX_DPS368_CFG_FIFO_DISABLE_VAL             0
+
+#define     IFX_DPS368_CFG_INTR_PRS_ENABLE_POS          4
+#define     IFX_DPS368_CFG_INTR_PRS_ENABLE_VAL          1U
+#define     IFX_DPS368_CFG_INTR_PRS_DISABLE_VAL         0U
+
+#define     IFX_DPS368_CFG_INTR_TEMP_ENABLE_POS         5
+#define     IFX_DPS368_CFG_INTR_TEMP_ENABLE_VAL         1U
+#define     IFX_DPS368_CFG_INTR_TEMP_DISABLE_VAL        0U
+
+#define     IFX_DPS368_CFG_INTR_FIFO_FULL_ENABLE_POS    6
+#define     IFX_DPS368_CFG_INTR_FIFO_FULL_ENABLE_VAL    1U
+#define     IFX_DPS368_CFG_INTR_FIFO_FULL_DISABLE_VAL   0U
+
+#define     IFX_DPS368_CFG_INTR_LEVEL_TYP_SEL_POS       7
+#define     IFX_DPS368_CFG_INTR_LEVEL_TYP_ACTIVE_H      1U
+#define     IFX_DPS368_CFG_INTR_LEVEL_TYP_ACTIVE_L      0U
+
+#define     IFX_DPS368_INTR_SOURCE_PRESSURE             0
+#define     IFX_DPS368_INTR_SOURCE_TEMPERATURE          1
+#define     IFX_DPS368_INTR_SOURCE_BOTH                 2
+
+#define     IFX_DPS368_INTR_STATUS_REG_ADDR             0x0A
+#define     IFX_DPS368_INTR_STATUS_REG_LEN              1
+
+
+#define     IFX_DPS368_INTR_DISABLE_ALL                (uint8_t)0b10001111
+
+
+
+/*=======================================================================
+
+MACROS INTERNAL DEFINITIONS
+
+========================================================================*/
+#define POW_2_23_MINUS_1	0x7FFFFF   //implies 2^23-1
+#define POW_2_24			0x1000000
```

Comment:
Again looks like tabs were used

Reply:
Coding style related issues are addressed in new commits and are already fixed now.